src/clinical/api/clinical-api.ts

```diff
@@ -186,15 +183,39 @@ class ClinicalController {
 
 		const donorEntityData = await service.getDonorEntityData(donorIds);
 
+		const entityRecords = donorEntityData[0].records;
```

Comment:
We need to check that `donorEntityData` has at least one item. `donorEntityData[0]` could be undefined.

Reply:
This one needs to be checked before we merge. 🙏

Reply:
Yes sorry, updating